samples/nrf9160/nrf_cloud_rest_device_message/README.rst

```diff
@@ -0,0 +1,92 @@
+.. _nrf_cloud_rest_device_message:
+
+nRF9160: nRF Cloud REST Device Message
+######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The REST Device Message sample demonstrates how to use the `nRF Cloud REST API`_ to send a `Device Message <nrf Cloud Device messages_>`_ using the SendDeviceMessage endpoint. The d2c topic is used in non-bulk mode.
```

Comment:
not sure about this formatting with the `<` `>`?

Reply:
It's a custom text link:
![custom link](https://user-images.githubusercontent.com/97475097/156257224-1d36127f-f9ec-4b41-88a3-f67444a3edc6.PNG)